settings.gradle

```diff
@@ -1,10 +1,4 @@
-rootProject.name = 'lightstep-tracer-java'
-
-include "java-common"
-project(":java-common").projectDir = file("common")
-
-include 'lightstep-tracer-jre'
-include 'examples:jre-simple'
+rootProject.name = 'lightstep-tracer-android'
 
 if (System.getenv()['ANDROID_HOME']) {
```

Comment:
these statements can be taken out of the `if` now that this repo is android only and whoever is building it should definitely have `ANDROID_HOME` defined.